.gitignore

```diff
@@ -36,5 +36,5 @@ yarn-error.log*
 next-env.d.ts
 
 # sqlite
-prisma/dev.db*
+prisma/db.dev
```

Comment:
You don't need to update this. The error is in `.env.example` file: the proper name is `dev.db` tho.

Please, use `dev.db` in your `.env` file and undo this change.

Reply:
Ok, I'll make the correction and I missed it too, to mark the winner of each round.